src/rest/CommunityEndpoints.js

```diff
@@ -23,39 +23,25 @@ export async function joinCommunity(communityAddress, memberAddress, secret = un
     return json
 }
 
-export async function withdraw(communityAddress, memberAddress, wallet) {
-    const json = await authFetch(
+export async function memberStats(communityAddress, memberAddress) {
+    const json = await fetch(
```

Comment:
is this really eslint-recommended style? Remove the `{}` on put on one line maybe. And instead of await, just return the Promise.